composer.json

```
@@ -41,39 +42,41 @@
         "elasticsearch/elasticsearch": "^7.17||^8.5",
         "ezyang/htmlpurifier": "^4.14",
         "guzzlehttp/guzzle": "^7.4",
-        "laminas/laminas-captcha": "^2.12",
-        "laminas/laminas-code": "^4.5",
-        "laminas/laminas-db": "^2.15",
-        "laminas/laminas-di": "^3.7",
+        "laminas/laminas-captcha": "^2.16",
```

What's not compatible with version `2.12` in this pull request?
```suggestion
        "laminas/laminas-captcha": "^2.12",
```